src/cohort/cohort_server.cc

```diff
@@ -0,0 +1,33 @@
+#include "src/cohort/cohort_server.h"
+
+#include "absl/status/statusor.h"
+#include "grpcpp/server_context.h"
+#include "src/proto/cohort.grpc.pb.h"
+#include "src/utils/grpc_utils.h"
+
+namespace cohort {
+
+using grpc::ServerContext;
```

Comment:
nit: `using ::grpc::ServerContext;` to align with google style and keeping local style consistent?

Reply:
Done.